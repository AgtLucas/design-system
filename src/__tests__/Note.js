import React from 'react'
import renderer from 'react-test-renderer'
import { Note, theme } from '..'

describe('Note', () => {
  test('renders', () => {
    const json = renderer.create(<Note />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
