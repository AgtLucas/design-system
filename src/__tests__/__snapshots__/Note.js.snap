// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note renders 1`] = `
.c0 {
  width: 100%;
  min-height: 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  font-size: 12px;
  color: #687B8E;
  background-color: #f6f8fa;
  border-radius: 2px;
  border-width: 1px;
  border-style: solid;
  border-color: #d1d6db;
  padding: 8px;
}

<div
  className="c0"
/>
`;
